Removes arrows from input in Firefox

min0k opened the pull request:
**Fixes Issue #14** 

This PR removes the arrows from the input in Firefox, so the design is consistent across Chrome and Firefox.

Before:
<img width="361" alt="Screen Shot 2022-05-03 at 5 18 23 PM" src="https://user-images.githubusercontent.com/91349690/166606110-0f43ae7f-416a-4b4a-8569-4631bc5da254.png">

After:
<img width="368" alt="Screen Shot 2022-05-03 at 5 18 29 PM" src="https://user-images.githubusercontent.com/91349690/166606112-0c15bceb-324d-4ed6-8769-bee00b08d5c8.png">

min0k committed:
Removes arrows from input in Firefox

min0k committed:
Undo Prettier changes

pg07codes commented:
@min0k  the issue seems to still persist in the [deploy preview](https://deploy-preview-17--fokusapp.netlify.app/). please check once. 
![image](https://user-images.githubusercontent.com/34238240/166654652-143a577b-71ee-4632-bbf1-1d9cbe861e19.png)

i am on ubuntu 18.04. what device are you using ? maybe that is the reason for discrepancy

min0k committed:
Remove arrows for other browsers

Previous commit targets Firefox, this commit targets Chrome, Safari, Edge, and Opera.

min0k commented:
@pg07codes I am on macOS Montery 12.0.1, using the latest version of Firefox. I don't think the operating system can influence how stuff gets rendered in the web browser (don't quote me on that though..)

I made another change targeting all browsers instead of just Firefox.

pg07codes commented on src/components/TaskBoard/TaskInput.js:
```diff
@@ -62,6 +62,14 @@ const TaskTimeInputDiv = styled.div`
         font-size: 1.2em;
         color: #fabb18;
     }
+    input[type="number"] { 
```
if you notice, i have already put the code to make the arrows disappear from webkit based browsers. the code is in TaskTimeInputField component (the one below which you have changed, line75,to be precise). i think putting the fix for firefox should there should work. As of now, in the deploy preview, i am seeing the arrow buttons

min0k replied:
Removed code that make arrows disappear from webkit based browsers (since you already added the code) and moved the firefox fix to TaskTimeInputDiv. This should ensure the fix is working in the deploy preview.

min0k committed:
Update TaskInput.js

pg07codes commented:
yes this seems to work fine. thanks.
can you please merge the commits into a single one, so i can merge the PR.

min0k committed:
Merge branch 'pg07codes:master' into uibug_arrows

min0k commented:
I believe you can squash and merge from your side - [GitHub Document](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/merging-a-pull-request).

I do not have write access to do this.

pg07codes commented:
thanks @min0k for the fix
